test/flink_test.go

```diff
@@ -120,3 +152,214 @@ func (s *CLITestSuite) TestFlink_Autocomplete() {
 		s.runIntegrationTest(test)
 	}
 }
+
+func (s *CLITestSuite) TestFlinkShell() {
+	tests := []FlinkShellTest{
+		{inputFile: "use-catalog", goldenFile: "use-catalog.golden"},
+		{inputFile: "use-database", goldenFile: "use-database.golden"},
+		{inputFile: "set-key", goldenFile: "set-key.golden"},
+		{inputFile: "reset-key", goldenFile: "reset-key.golden"},
+		{inputFile: "reset-all", goldenFile: "reset-all.golden"},
+	}
+
+	for _, test := range tests {
+		s.runFlinkShellTest(test)
+	}
+}
+
+func (s *CLITestSuite) runFlinkShellTest(flinkShellTest FlinkShellTest) {
+	s.T().Run(flinkShellTest.inputFile, func(t *testing.T) {
+		s.login(t)
+
+		// Create a file for go-prompt to use as the input stream
+		stdin, err := os.Create("input.txt")
```

Comment:
```suggestion
		stdin, err := os.Create("flink_shell_input_stream.txt")
```
maybe reduce the chance that the CI starts to fail because someone created a input.txt file here? lol